visual-java/src/main/java/com/saucelabs/visual/model/FullPageScreenshotConfig.java

```diff
@@ -105,4 +117,12 @@ public int getScrollLimit() {
   public void setScrollLimit(int scrollLimit) {
     this.scrollLimit = scrollLimit;
   }
+
+  public String getScrollElement() {
```

Comment:
I think it would be better if we rename the method to getScrollElementId

Reply:
it's used by json builder so it has to match the name in spec